### PR TITLE
adds the ability do db:migrate:down for the db view.

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

**Comment by isimluk** [comment]
Merged as b0f14256d286b38b86dbb6d22c736f10f2aa60f0. Thank You!

File: db/migrate/20141121164042_replace_arf_report_breakdown_view.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 class ReplaceArfReportBreakdownView < ActiveRecord::Migration
-   def self.up
-     execute <<-SQL
-CREATE OR REPLACE VIEW scaptimony_arf_report_breakdowns AS
+  def self.up
+    CreateScaptimonyArfReportBreakdowns.down
+    execute "DROP VIEW scaptimony_arf_report_breakdowns"
```
Comment (ohadlevy) [inline db/migrate/20141121164042_replace_arf_report_breakdown_view.rb]: damn, this was typo leftover, PR incoming

Comment (isimluk) [inline db/migrate/20141121164042_replace_arf_report_breakdown_view.rb]: I don't follow, what's exactly the typo?

Comment (ohadlevy) [inline db/migrate/20141121164042_replace_arf_report_breakdown_view.rb]: i think i pushed to my branch at the same time you were merging.
i guess you dont have the latest version from my branch, but i don't think its an issue.

**Pull request closed without merging**